iosApp/iosApp/ContentView.swift

```diff
@@ -167,6 +167,12 @@ struct ContentView: View {
                         nearbyVM.goBack()
                     }
                 }, content: coverContents)
+                .onAppear {
+                    viewportProvider.updateCameraState(locationDataManager.currentLocation)
```

Comment:
issue: it looks like the 2nd AC isn't addressed here - if I pan the map & then turn on hide maps, I am stuck at that panned location

Reply:
Oh hm, I thought I tried to reproduce that and couldn't after the fix 🤔

Reply:
Oh, I think this might only happen when location services is turned off. But if it is, then it's unclear what we would want to do instead, since we have no way to get a different location.

Reply:
Oh looks like it only happens when location services is turned off. That may be fine behavior, in coordination with [showing the location services button](https://github.com/mbta/mobile_app/pull/510/files#r1826047141).